advanced/deploying_deno/kinsta.md

```diff
@@ -27,8 +27,8 @@ To do so, your `package.json` should look like this:
 ## Example application
 
 ```js
-import { serve } from "https://deno.land/std@0.168.0/http/server.ts";
```

Comment:
I think it would be better to use `https://deno.land/std@$STD_VERSION/http/server.ts` which gets mapped to the latest version of `std`. Same for the other imports that use specific `std` versions.